Palindrome.UnitTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+// Added a test for Empty Input.
+// Created branch 'review' for PR sharing.
+
+
+namespace PalindromeArray.UnitTests
+{
+    [TestClass]
+    public class PalindromeUnitTests
```

Comment:
You are missing a test case for null.

Reply:
Is this something you do in code generally? Null check everything?

Reply:
yep! check out the ?? operator and ?: operator in C# 
Imagine you request information from the server using Ajax, and you expected the value to populate your web page...but the server is slow to respond so when the page gets loaded each value is null. How do you handle those situations when you are expecting data, but an unreliable system might not have retrieved that data yet? Graceful exception? Try Catch? Null check? await/async?

Reply:
All of @rileymckenna's suggestions are always valid.

> **RULE OF THUMB:** Don't trust inputs to any public methods unless you are guaranteed about the format for some reason.

If you have a method that accepts `int`, you know it *must* be a number, but you have no guarantees about the range of the number. This is why you must ensure that you are validating parameters before making assumptions about them. Aside from the stability of your code, you also want to make sure data is validated for security reasons (e.g. SQL injection attacks).